src/main/java/ch/njol/skript/expressions/ExprDurability.java

```diff
@@ -39,34 +43,51 @@
 /**
  * @author Peter Güttinger
  */
-@Name("Data/Damage Value")
-@Description({"The data/damage value of an item/block. Data values of blocks are only supported on 1.12.2 and below.",
-		"You usually don't need this expression as you can check and set items with aliases easily, ",
-		"but this expression can e.g. be used to \"add 1 to data of &lt;item&gt;\", e.g. for cycling through all wool colours."})
+@Name("Data/Damage Value/Durability")
+@Description({"The data/damage value/durability of an item/block. Data values of blocks are only supported on 1.12.2 and below.",
+	"You usually don't need this expression as you can check and set items with aliases easily, ",
+	"but this expression can e.g. be used to \"add 1 to data of &lt;item&gt;\", e.g. for cycling through all wool colours."})
 @Examples({"set damage value of player's tool to 10",
-		"set data value of target block of player to 3",
-		"add 1 to the data value of the clicked block",
-		"reset data value of block at player"})
+	"set data value of target block of player to 3",
+	"add 1 to the data value of the clicked block",
+	"reset data value of block at player",
+	"set durability of player's held item to 0"})
 @Since("1.2")
 public class ExprDurability extends SimplePropertyExpression<Object, Long> {
 
+	private static final boolean LEGACY_BLOCK = !Skript.isRunningMinecraft(1, 13);
+	private boolean durability;
+
 	static {
-		register(ExprDurability.class, Long.class, "((data|damage)[s] [value[s]]|durabilit(y|ies))", "itemtypes/blocks/slots");
+		register(ExprDurability.class, Long.class, "((data|damage)[s] [value[s]]|durability:durabilit(y|ies))", "itemtypes/blocks/slots");
+	}
+
+	@Override
+	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, SkriptParser.ParseResult parseResult) {
+		durability = parseResult.hasTag("durability");
+		setExpr(exprs[0]);
+		return true;
 	}
-	
+
 	@Override
 	@Nullable
 	public Long convert(final Object o) {
 		if (o instanceof Slot) {
 			final ItemStack i = ((Slot) o).getItem();
-			return i == null ? null : (long) ItemUtils.getDamage(i);
+			if (i != null) {
+				long damage = ItemUtils.getDamage(i);
+				return durability ? i.getType().getMaxDurability() - damage : damage;
+			}
 		} else if (o instanceof ItemType) {
 			ItemStack item = ((ItemType) o).getRandom();
-			return (long) ItemUtils.getDamage(item);
+			long damage = ItemUtils.getDamage(item);
+			return durability ? item.getType().getMaxDurability() - damage : damage;
+		} else if (LEGACY_BLOCK && o instanceof Block) {
```

Comment:
If you do this first you can extract a single `ItemStack` variable and then do the damage bit together.
```
final ItemStack stack;
if (o instanceof ItemType) stack = ...
else if (o instanceof Slot) stack = ...
else return null;
long damage = ...
return durability ? item.getType().getMaxDurability() - damage : damage;
```
This should make the method shorter and the code easier to read :)